recipes/xtl/all/conanfile.py

```diff
@@ -79,6 +80,7 @@ def _module_file_rel_path(self):
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "xtl"
         self.cpp_info.names["cmake_find_package_multi"] = "xtl"
+        self.cpp_info.set_property("cmake_target_name", "xtl")
```

Comment:
I belive the current move is to make these targets name absolute, so xtl::xtl in this case (?)

Reply:
There's usually a delay, so it might be worth holding off but this does benefit some consumers

Reply:
I agree with you.
I made this PR draft.
When CI update conan 1.43.0 or later, I make it "ready for review".

Reply:
It's time to fix.